test/core/binary-leb128.wast

```diff
@@ -33,9 +33,10 @@
   "\00asm" "\01\00\00\00"
   "\04\04\01"                          ;; Table section with 1 entry
   "\70\00\00"                          ;; no max, minimum 0, funcref
-  "\09\07\01"                          ;; Element section with 1 entry
+  "\09\09\01"                          ;; Element section with 1 entry
```

Comment:
Just so that we're clear: this is doing the 2nd alternative

> or at least making an explicit decision about this backward-incompatible binary representation change and removing (?) corresponding tests

from my issue description, right?

Because this changes the test itself rather than the spec, it basically acknowledges that new layout is a breaking change, but we think it's fine and won't break anything in the wild?

Reply:
@gahaas @binji Ping. Was there any resolution here?

Reply:
Since this change was already made in https://github.com/WebAssembly/reference-types/pull/95, it makes sense to do the same here.